src/sql/workbench/api/common/sqlExtHost.api.impl.ts

```diff
@@ -402,8 +402,14 @@ export function createAdsApiFactory(accessor: ServicesAccessor): IAdsExtensionAp
 				createWebViewDialog(name: string) {
 					return extHostModalDialogs.createDialog(name);
 				},
-				createModelViewDialog(title: string, dialogName?: string, isWide?: boolean): azdata.window.Dialog {
-					return extHostModelViewDialog.createDialog(title, dialogName, extension, !!isWide);
+				createModelViewDialog(title: string, dialogName?: string, width?: boolean | azdata.window.DialogWidth): azdata.window.Dialog {
```

Comment:
Maybe put a comment here explaining that the boolean is for backwards compat support